### ISSUE TITLE
Container query syntax support

### ISSUE THREAD

**Issue by woody-li** [clean-css/clean-css#1232]
CSS has defined container query.
https://www.w3.org/TR/css-contain-3/#container-queries

But it seems clean-css doesn't support it.

Is is possible to support it or any plugin / workaround for it?

### Input CSS

```css
@container (width<1024px) {
    .test {
        color: #fff;
    }
}
```

### Actual output CSS

- Nothing by default options.

- Disable remove empty rules and nested blocks : 
```css
@container (width<700px){}
```

- Only enable level 0 (end bracket missing):
```css
@container (width<700px){.test {color:#fff}
```

### Expected output CSS

```css
@container (width<700px){.test{color:#fff;}}
```

**Comment by calvinjuarez** [comment]
`@layer` may have a similar issue.

**Comment by sevikerr** [comment]
Same issue! Any news?

**Comment by jakubpawlowicz** [comment]
The fix is out in 5.3.2.
